src/live/index.js

```diff
@@ -25,6 +26,21 @@ const onConnect = (io, socket) => {
   socket.on('chat:album', async (data) => {
     handleAlbumChat(io, socket, data);
   });
+
+  socket.on('listening-to-album-play', (albumId) => {
+    albumController.startAlbumPlayback(albumId);
```

Comment:
Currently this function doesn't exist; what are you planning on having the controller do? So far in the ticket outlines, we will just be emitting a message back to the frontend so that other users can see what this user is listening to; or we can list all the users who are listening to this album right now. But if you have additional functionality to add, I'm all for it!

Reply:
Actually I was thinking about to suggest similar albums or tracks to users based on their listening history or preferences. But possibly it should a part of this ticket, I don't know.

Reply:
Ah I see, that's really cool! Spotify even has an API for that :open\_mouth: : [https://developer.spotify.com/documentation/web-api/reference/get-recommendations](https://developer.spotify.com/documentation/web-api/reference/get-recommendations). I can look into it a bit more and write a ticket around that. 
  
For now, we can just comment out this bit though since it will cause errors as it is since there is no function called `startAlbumPlayback` being exported from that controller file.

Reply:
sounds good 👍  You could also leave the event listener in and just comment out line 31; that way you coould still have a console.log line here to test that the connection between the frontend and backend is working for the play button.

Reply:
I have removed albumId as parameter due to eslint problem. As albumId is declared but never used. commit 7c301b1032f5bb7d211517d360af28dc600fe5bc